### ISSUE TITLE
Page experience: FID issue possibly caused by @ampproject/pull/34028

### ISSUE THREAD

**Issue by erwinmombay** [ampproject/wg-performance#85]
Details
---

Hello,
I work on a few sister sites that share the same code base and AMP template. Since May 28th they have been experiencing Core Web Vitals issues on AMP pages, specifically First Input Delay. However, we haven’t made any code changes to AMP recently and these are blog pages there are no interactive features that should trigger FID to be measured.
The only user interactive function I can think of is that images on AMP have a lightbox feature but this is bundled into AMP by default. I checked the AMPHTML changelogs and there is a change to AMP images and lightbox at the end of April. CrUX data has a 28-day collection period so seeing the issues being reported at the end of May lines up with this change.
Can any AMP devs comment on the possibility this change being related to FID issues? and would be able to investigate/report it to the appropriate channel?

---

Components in use: 
"amp-bind", "amp-ad", "amp-analytics", "amp-social-share", "amp-iframe", "amp-youtube", "amp-twitter", "amp-instagram", "amp-facebook", "amp-brightcove", "amp-sidebar", "amp-sticky-ad", "amp-video", "amp-fx-collection", "amp-auto-lightbox", "amp-lightbox-gallery"

/cc @ampproject/wg-performance

**Comment by erwinmombay** [comment]
@issababble had a few questions:

- "Since May 28th they have been experiencing Core Web Vitals issues on AMP pages" - is this still the case? we saw a slight dip of our FID performance in april and may 2021 but have been seeing a large improvement since june (going to figure out what we changed for the improvement). I just want to line up the dates to narrow down.

- is https://bit.ly/3woT5V4 still a good example page that demonstrates the issue?

**Comment by issababble** [comment]
@erwinmombay Yes I still see "needs improvement" FID pages for two sites I work on (same AMP templates as they are sister sites). On one site, it started May 28, and on the other site it started June 9. The pages have not decreased on either site since they began. 

However, I know Google groups pages that may not have field data so I did a quick analysis of a sample of 21 pages from each site (these are the pages reported in Search Console). Out of the 42 total, there are 7 pages that are still having FID issues in the field data. There is 1 page that is now green/good for FID. The rest of them no longer have field data. So it's possible there have been slight improvements but 7 pages out of my sample have had issues in the last 28 days. 

Yes, that page is one of the 7 and it gets consistent visits so it should continue to be a good example. Here is another one from the sister site: https://bit.ly/3fzqu94 

If you think of other questions, I'm happy to answer!
